modules/wkt/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@loaders.gl/wkt",
+  "description": "Loader and Writer for the WKT (Well Known Text) Format",
+  "version": "0.0.1",
```

Comment:
Shouldn't be 2.0.0?

Reply:
Yes, I was assuming lerna would adjust it automatically when we publish...

As to version number, I am not sure. I believe you have published 2.0.0 already? If so maybe we put this loader in 2.1.0-alpha.1?